set.mm

```diff
@@ -775935,4 +775936,303 @@ to Davis and Putnam (1960).  (Contributed by David A. Wheeler,
+      UHUEOPQRUDUGKZUFUISAUMUEUHJUDUGUCTUBRUA $.
+
+    $d J x y z $.  $d c x y z $.  $d d x y z $.  $d ph x y z $.
+    $( Open neighborhood and neighborhood is equivalent regarding disjointness
+       for both sides.  Lemma for ~ isnrm4 .  (Contributed by Zhi Wang,
+       1-Sep-2024.) $)
+    isnrm4lem3 $p |- ( ph ->
+    ( E. x e. ( ( nei ` J ) ` c ) E. y e. ( ( nei ` J ) ` d ) ( x i^i y ) = (/)
+    <-> E. x e. J E. y e. J ( c C_ x /\ d C_ y /\ ( x i^i y ) = (/) ) ) ) $=
+      ( vz cv wss cin c0 wceq cnei cfv wrex wa w3a isnrm4lem2 rexbidv anbi2d wi
+      ssrin sseq0 ex adantl reximdv simpr ineq1d eqeq1d opnneieqv wb isnrm4lem1
+      syl a1i 3bitr4d ) AEIZBIZJZURCIZKZLMZCFIZDNOZOZPZQZBDPUSVCUTJZVBQCDPZQZBD
+      PZVFBUQVDOPUSVHVBRCDPBDPZAVGVJBDAVFVIUSABCDFGSUATAVFHIZUTKZLMZCVEPBHUQDGA
+      VMURJZQVBVOCVEVPVBVOUBZAVPVNVAJZVQVMURUTUCVRVBVOVNVAUDUEUNUFUGAURVMMZQZVB
+      VOCVEVTVAVNLVTURVMUTAVSUHUIUJTUKVLVKULAUSVHVBBCDUMUOUP $.
+  $}
+
+  ${
+    $d J c d x y $.
+    $( A topological space is normal iff any two disjoint closed sets are
+       separated by neighborhoods.  (Contributed by Zhi Wang, 1-Sep-2024.) $)
+    isnrm4 $p |- ( J e. Nrm <-> ( J e. Top /\ A. c e. ( Clsd ` J ) A. d e. (
+    Clsd ` J ) ( ( c i^i d ) = (/) -> E. x e. ( ( nei ` J ) ` c ) E. y e. ( (
+    nei ` J ) ` d ) ( x i^i y ) = (/) ) ) ) $=
+      ( cnrm wcel ctop cv cin c0 wceq wss w3a wrex wi ccld cfv wral wa 2ralbidv
+      cnei isnrm3 id isnrm4lem3 imbi2d pm5.32i bitr4i ) CFGCHGZDIZEIZJKLZUJAIZM
+      UKBIZMUMUNJKLZNBCOACOZPZECQRZSDURSZTUIULUOBUKCUBRZROAUJUTROZPZEURSDURSZTA
+      BCDEUCUIVCUSUIVBUQDEURURUIVAUPULUIABCDEUIUDUEUFUAUGUH $.
+  $}
+
+  ${
+    $d c d j x y $.
+    $( A topological space is normal if any disjoint closed sets can be
+       separated by open neighborhoods.  An alternate definition of ~ df-nrm .
+       (Contributed by Zhi Wang, 30-Aug-2024.) $)
+    dfnrm2 $p |- Nrm = { j e. Top | A. c e. ( Clsd ` j ) A. d e. ( Clsd ` j )
+                 ( ( c i^i d ) = (/) -> E. x e. j E. y e. j ( c C_ x /\ d C_ y
+                 /\ ( x i^i y ) = (/) ) ) } $=
+      ( cnrm cv ctop wcel cin c0 wceq wss w3a wrex wi ccld cfv wral wa cab crab
+      isnrm3 abbi2i df-rab eqtr4i ) FCGZHIDGZEGZJKLUHAGZMUIBGZMUJUKJKLNBUGOAUGO
+      PEUGQRZSDULSZTZCUAUMCHUBUNCFABUGDEUCUDUMCHUEUF $.
+
+    $( A topological space is normal if any disjoint closed sets can be
+       separated by neighborhoods.  An alternate definition of ~ df-nrm .
+       (Contributed by Zhi Wang, 2-Sep-2024.) $)
+    dfnrm3 $p |- Nrm = { j e. Top | A. c e. ( Clsd ` j ) A. d e. ( Clsd ` j )
+    ( ( c i^i d ) = (/) -> E. x e. ( ( nei ` j ) ` c ) E. y e. ( ( nei ` j ) `
+    d ) ( x i^i y ) = (/) ) } $=
+      ( cnrm cv ctop wcel cin c0 wceq cnei cfv wrex wi ccld wral wa cab isnrm4
+      crab abbi2i df-rab eqtr4i ) FCGZHIDGZEGZJKLAGBGJKLBUHUFMNZNOAUGUINOPEUFQN
+      ZRDUJRZSZCTUKCHUBULCFABUFDEUAUCUKCHUDUE $.
+  $}
```

Comment:
It's nice to have looped back to definitions in the form `Nrm =`, however the most useful form are the `isnrm4`-like  ones and I think in this case that effort was not necessary.

Reply:
I agree. But I thought that `isnrm4` and things alike are also for pedagogical uses so it would be great to have them included in description of `df-nrm`. Can we do it with only `isnrm4` and without `dfnrm3`?

Reply:
The `isnrm4` version is easier to understand than `dfnrm3`, and often in the comment of the `df-xxx` definitions we point to the `isxxx` or `xxxval` versions for that reason.

Anyway, nothing wrong with adding them!